LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2023-2024 Justus Perlwitz
```

Comment:
Is this intended? I wouldn't normally expect an open source contribution to take over copyright.